src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -11,6 +11,8 @@ namespace System.Net.Tests
 {
     public class HttpListenerPrefixCollectionTests
     {
+        public static bool IsNonZeroLowerBoundArraySupported { get; } = PlatformDetection.IsNonZeroLowerBoundArraySupported;
```

Comment:
Nit: AFAIK, [expression-bodied properties](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties#expression-body-definitions) are preferred. That is, using `=>` to implement a read-only property instead of doing `{ get; } =`. @davidsh @CIPop  can correct me if I'm wrong.

Same comment applies to all new `{ get; } =` being added.